lib/engine/game/g_1824_cisleithania/game.rb

```diff
@@ -453,25 +459,37 @@ def select_randomly(collection)
         def stackify(company, header)
           return header unless company.stack
 
-          real_header = "#{header} STACK #{company.stack}"
-          real_header += ' (CR)' if company.stack == 1
-          real_header
+          case company.stack
+          when 1
+            'Construction Railway (Stack 1)'
+          when 2
+            'Large Pre-staatsbahn (Stack 2)'
+          when 3
+            'Small Pre-staatsbahn (Stack 3)'
+          when 4
+            'Coal Railway (Stack 4)'
+          end
         end
 
         def closed_construction
           @close_construction_company_when_first_5_sold ? '5' : '4'
         end
 
-        def create_construction_railway_from_bought_pre_staatsbahn(company, minor)
-          make_minor_construction_railway(minor)
+        def make_minor_construction_railway(minor)
+          @log << "#{minor.name} becomes a construction railway. "\
+                  'It returns its cash to the bank as it does not use any money.'
+          minor.spend(minor.cash, @bank)
```

Comment:
Is it possible that the minor will have no cash? You will get an error.

Reply:
The pre-staatsbahn gets the price paid as starting money. This is the pre-staatsbahn drafted from stack 1 during the 4 first turns in SR1.